blog/urls.py

```diff
@@ -2,11 +2,14 @@
 from django.conf.urls import include, url
 from django.conf.urls.static import static
 from django.contrib import admin
+from posts.views import MyView, post_update
 
 urlpatterns = [
     url(r'^$', 'posts.views.home', name='home'),
     # url(r'^posts/$', MyView.as_view(), name='posts'),
     url(r'^posts/', include('posts.urls')),
+    url(r'^posts/(?P<id>[0-9]+)/edit/$', 'posts.views.post_update', name='update'),
+    url(r'^posts/(?P<id>[0-9]+)/detail/$', 'posts.views.post_detail', name='detail'),
```

Comment:
I suggest you move these two into `posts.urls` module to have every url beginning with `posts` all together in the same module.

Reply:
Also, I think these `name='update'` and `name='detail'` are generic. You may want to go with `name='post_update'` and `name='post_detail'` to avoid being forced to change it later in the future when there's another resource that also has update and details. Let me know if this doesn't make sense.

Reply:
Thanks. I will work on all the feedback.